swagger/v1/journey_event_name_attribute.json

```diff
@@ -0,0 +1,13 @@
+{
+  "JourneyEventName": {
+    "type": "string",
+    "example": "complete",
+    "enum": [
+      "complete",
+      "uncomplete",
```

Comment:
Maybe consider `resume` or `partial` rather than `uncomplete` ? What state does a completed journey move to when it gets uncompleted?

Reply:
hi @smoothcontract it moves back to `in_progress` (defined in the statemachine). I'd rather not change these enums now as they have already been reviewed in earlier PRs.

Reply:
Okay 👍